INSTALL.md

```diff
@@ -21,13 +21,51 @@ psql -d gis -f indexes.sql
 Additionally you need some shapefiles.
 
 ## Scripted download
+To download the shapefiles you can run the following script. No further steps should be needed as the data has been processed and placed in the requisite directories.
 
-To download the shapefiles you can run the following script from this directory. No further steps should be needed as the data has been processed and placed in the requisite directories.
+```
+scripts/get-shapefiles.py
+```
+
+This script generates and populates the *data* directory with all needed
+shapefiles, including indexing them through *shapeindex*.
 
+Command-line syntax:
 ```
-./get-shapefiles.sh
+usage: get-shapefiles.py [-h] [-c] [-d <directory name>] [-e] [-f] [-l] [-n]
+                          [-p] [-r] [-s] [-u] [-1] [-2] [-3] [-4] [-5] [-6]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -c, --check           check whether the 'data' directory already exists
+  -d <directory name>, --directory <directory name>
+                        set the name of the data directory (default: 'data')
+  -e, --no-extract      do not populate target directories with the expansion
+                        of downloaded data
+  -f, --force           force continuing even if project.yaml does not exist
+  -l, --no-curl         do not use 'curl' even if available
+  -n, --no-download     do not download archive if already existing locally
+  -p, --pause           pause before starting
+  -r, --remove          remove each downloaded archive after its expansion
+  -s, --no-shape        do not run shapeindex
+  -u, --update          force downloading files even if not newer than the
+                        locally existing ones
+  -1, --world-boundaries
+                        only process world_boundaries
+  -2, --simplified-land
+                        only process simplified-land-polygons-complete-3857
+  -3, --ne-admin        only process ne_110m_admin_0_boundary_lines_land
+  -4, --land-polygons   only process land-polygons-split-3857
+  -5, --icesheet-polygons
+                        only process antarctica-icesheet-polygons-3857
+  -6, --icesheet-outlines
+                        only process antarctica-icesheet-outlines-3857
```

Comment:
Ok, I'll remove them. Also, they could be reported here https://github.com/gravitystorm/openstreetmap-carto/pull/2333